### PR TITLE
fix config map for readiness probe

### PR DESCRIPTION
**Pull request by vegichan**
The Nuodb controller was not reconciling config maps

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -69,8 +69,7 @@ func main() {
 
 	namespace, err := k8sutil.GetWatchNamespace()
 	if err != nil {
-		log.Error(err, "Failed to get watch namespace")
-		os.Exit(1)
+		namespace = "default"
```
Comment (thebithead) [inline cmd/manager/main.go]: What is the reason for this change?

Comment (vegichan) [reply cmd/manager/main.go]: there is no need to create a namespace. The operator is capable of watching the `default` namespace by default.

Comment (thebithead) [reply cmd/manager/main.go]: There is nothing here that creates a K8s namespace.  It up the the customer to decide what K8s namespace they want to use.  

Our operator is deployed at namespace scope.

Line 70 gets the namespace that the current operator is watching.  If out Operator cannot determine the namespace that the operator is watching, then it should fail and exit immediately.

Comment (vegichan) [reply cmd/manager/main.go]: Providing a default by default is a really bad pattern. Why should I run `operator-sdk run --local --namespace=default` if I can run without a namespace? But it is not important enough to have this conversation. I reverted the change.
